content/docs/06-integrations/00-kubernetes-dashboard.md

```diff
@@ -17,92 +17,604 @@ import Tooltip from "shared/components/ui/Tooltip";
+The [Kubernetes Dashboard](https://github.com/kubernetes/dashboard) add-on pack is a general-purpose, web-based UI that allows you to manage clusters and the applications deployed in them. 
 
-The Kubernetes Dashboard pack you can use to access the dashboard requires the Spectro Proxy pack to expose the dashboard.
-<br /> 
+<br />
+
+## Versions Supported
+
+<Tabs>
+
+<Tabs.TabPane tab="2.7.x" key="2.7.x">
 
-<InfoBox>
+## Prerequisites
 
-Palette supports provisioning a reverse proxy that can expose the Kubernetes dashboard. Use the [Spectro Proxy](/integrations/frp) pack to enable this capability. 
+- Outbound internet connectivity for port 443 is allowed so that you and your applications can connect with the [Spectro Proxy](/integrations/frp) reverse proxy.
 
-</InfoBox>
 
-A user-friendly version of this pack is available, Spectro Kubernetes Dashboard, which requires minimal configuration when used with the default settings. To learn more, check out [Spectro Kubernetes Dashboard](/integrations/spectro-k8s-dashboard).
+- Users or groups must be mapped to a Kubernetes RBAC role, either a Role or a ClusterRole. You can create a custom role through a manifest and use Palette's roleBinding feature to associate the users or groups with the role. Refer to the [Create Role Bindings](/clusters/cluster-management/cluster-rbac#createrolebindings) guide to learn more.
+
+## Parameters
+
+| Name | Supported Values | Default Values | Description |
+| --- | --- | --- | --- |
+| `k8s-dashboard.namespace` | | kubernetes-dashboard | The namespace where you want to run the Kubernetes Dashboard deployment |
+| `k8s-dashboard.clusterRole` | Any valid clusterRole from the Kubernetes cluster | `k8s-dashboard-readonly` | The ClusterRole to be attached to the ServiceAccount which  defines RBAC to the cluster resources.<br /><br />By default, a ClusterRole (k8s-dashboard-readonly) with ReadOnly access to most of the resources is deployed. |
+| `k8s-dashboard.certDuration` | A Go time.Duration string format in s (seconds), m (minutes), and h (hour) suffixes | `8760h` (365 days) | Validity for the Self-signed certificate, specified in hours. |
+| `k8s-dashboard.certRenewal` | A Go time.Duration string format in s (seconds), m (minutes), and h (hour) suffixes | `720h` (30 days) | Certificate renew before expiration duration |
+| `k8s-dashboard.serviceType` | ClusterIP, LoadBalancer | ClusterIP | The ServiceType for dashboard. <WarningBox>It is highly recommended to use ClusterIP service type to restrict access to the cluster</WarningBox> |
+| `k8s-dashboard.skipLogin` | True, False | False | A flag to skip authentications in the Kubernetes Dashboard UI. <WarningBox> Enabling this might expose a security risk. Use this only for demo purposes.</WarningBox> |
+

```

Comment:
A couple of notes on this one @ritawatson.  We have a bit too many warning boxes, I think we can simply put the message in the description. Also, when we are recommending something, let's use "we recommend", otherwise it can feel like we are hiding. Let's own up to our recommendations 😄 

I'm also not sure we want to ever put these types of boxes inside a table, but that could be a good team discussion topic.

![CleanShot 2023-06-22 at 09 35 19](https://github.com/spectrocloud/librarium/assets/29551334/dfcbbc69-3875-4df7-b273-e36ae59e1ccf)

Reply:
Removed warning boxes and revised the statements.

Reply:
Removed warning boxes from table and made them statements (with revised wording).